qri/client.py

```diff
@@ -25,7 +25,7 @@ def list():
 
 def get(ref):
     """get a dataset in the repository by reference"""
-    cmd = 'qri get --format json %s' % ref
+    cmd = ['qri', 'get', '--format', 'json', ref]
```

Comment:
One downside is that if we ever omit a comma, such as: `'qri' 'get'`, it would silently compile (concatenating the strings). Maybe better as multi-line?